sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml

```diff
@@ -3,27 +3,19 @@ services:
   pgsql:
     image: "postgres:14.3"
     container_name: pgsql
+    restart: unless-stopped
     ports:
       - "5432:5432"
     environment:
-      - POSTGRES_PASSWORD=Test1234
-    restart: unless-stopped
+      - POSTGRES_PASSWORD=Test1234    
     volumes:
       - pgsqldata:/var/lib/postgresql/data
       #- /var/lib/smartface/pgsql:/var/lib/postgresql/data
 
-  pgadmin:
```

Comment:
we do not want pgadmin no more ?

Reply:
Actually I wanted just to save some performance on jetson, however I can rollback if you guys insist on this

Reply:
AFAIK pgadmin does not have an arm image, so for jetson the container could not start properly either way... so removing it is OK